test/jdk/javax/swing/JEditorPane/8080972/TestJEditor.java

```diff
@@ -39,15 +39,11 @@
  * @bug 8080972 8169887
  * @summary Audit Core Reflection in module java.desktop for places that will
  *          require changes to work with modules
- * @author Alexander Scherbatiy
- * @run main/othervm -Djava.security.manager=allow TestJEditor
+ * @run main TestJEditor
  */
 public class TestJEditor {
 
     public static void main(String[] args) throws Exception {
-
-        SwingUtilities.invokeAndWait(TestJEditor::testJEditorPane);
-        System.setSecurityManager(new SecurityManager());
         SwingUtilities.invokeAndWait(TestJEditor::testJEditorPane);
     }
 
```

Comment:
Is there any need to catch the exception and rethrow RuntimeException below? I think we can remove try-catch block altogether...

Reply:
Updated

Reply:
Fixed in https://github.com/openjdk/jdk/pull/21498/commits/d9ee496f7349cb8beaf1e696fd430f8064baee8e